wallet/src/actors/app/handlers/create_data_req.rs

```diff
@@ -3,32 +3,52 @@ use serde::{Deserialize, Serialize};
 
 use crate::{
     actors::app,
-    types::{self, Hashable as _, ProtobufConvert as _},
+    types::{
+        self, from_generic_type, into_generic_type, number_from_string, u64_to_string,
+        DataRequestOutputHelper, Hashable as _, ProtobufConvert as _, TransactionHelper,
+    },
+};
+pub use witnet_data_structures::{
```

Comment:
`pub use` means "import and re-export", we only need to import:

```suggestion
use witnet_data_structures::{
```